index.bs

```diff
@@ -5783,8 +5796,8 @@ failed</dfn> steps given |navigable| and |navigation status|:
 </dl>
 
 <div algorithm>
-The [=remote end event trigger=] is the <dfn export>WebDriver BiDi user prompt
-closed</dfn> steps given |window|, |type|, |accepted| and optional |user text|
+The [=remote end event trigger=] is the <dfn export>WebDriver BiDi user prompt closed</dfn> steps
+given {{Window}} |window|, string |type|, boolean |accepted| and optional text |user text|
```

Comment:
If we are going to use the actual type for arguments we still have a lot of other algorithms which don't have them yet. Shall we maybe update them at once?

Reply:
I don't think it is critical, and can be done (or not) later. The scope of this change is to provide types only to the externally-exposed hooks.